src/mem/cache/base.hh

```diff
@@ -266,6 +266,13 @@ class BaseCache : public ClockedObject
 
         bool isBlocked() const { return blocked; }
 
+        void schedTimingResp(PacketPtr pkt, Tick when) override {
+            // Do when + 1 to avoid misschedule.
+            // This is the case when several schedules have to be carried out
+            // in the same cycle
+            QueuedResponsePort::schedTimingResp(pkt, when + 1);
+        }
```

Comment:
I wouldn't make the method virtual simply to add +1 to one of the parameters. If we really want to do this, we should just call schedTimingResp with when+1

Reply:
I agree.